src/App.js

```diff
@@ -36,8 +43,9 @@ function AppNavbar() {
           />{" "}
         </Link>
       </Navbar.Brand>
-      <Nav className="ml-auto text-muted">
-        {/* <div>Help 1200+ people</div> */}
+      <Nav className="ml-auto">
+        <Link to="/about">About</Link>
+        <Link to="/update">Add location</Link> ·{" "}
```

Comment:
add location -> add store

Reply:
also cut out the ` ·{" "}`

Reply:
this link needs to go to `/location` or to `/update`. see `getLinkTo()` in the code.